src/app/sign-in.tsx

```diff
@@ -0,0 +1,30 @@
+import { useRouter } from 'expo-router';
+import { showMessage } from 'react-native-flash-message';
+
+import { useLogin } from '@/api/auth/use-login';
+import type { LoginFormProps } from '@/components/login-form';
+import { LoginForm } from '@/components/login-form';
+import { useAuth } from '@/core';
+import { FocusAwareStatusBar } from '@/ui';
+
+export default function Login() {
+  const router = useRouter();
+  const signIn = useAuth.use.signIn();
+  const { mutate: login } = useLogin({
+    onSuccess: (data) => {
+      signIn({ access: data.accessToken, refresh: data.refreshToken });
```

Comment:
Don't you need an `await` here?

Reply:
not actually, this is the action should just update the store